.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish NPM Package
+
+# Automated publishing to NPM, using NPM_TOKEN GitHub secret
+# For more details, refer to https://github.com/amazon-connect/amazon-connect-chatjs/pull/164
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment:
+      name: release
+      url: https://www.npmjs.com/package/amazon-connect-chatjs
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
This is great, Is there a way to prompt for manual intervention here if during `npm i` or `npm run release` - new dist or package-lock files are created ?

Reply:
I haven't looked into manual confirmation for middle step. What are the exact concerns? Developer will manually create/draft a repository "Release", and then they also have to manually approve the workflow.

Ideally `package-lock` should be up to date on master already. Best practice would be not committing `dist` since it will always create merge conflicts...

This workflow will run in Node 16, so that will be the source of truth. I know there have been issues with the `bundle.map.js` files as well, I think these just differ between versions of node. 

@mhiaror - adding notes here, but will bring to team in post-standup so everyone is onboard.

Reply:
Published a revision - added a `npm publish --dry-run` and `git status` check, and admin can cancel/approve the workflow after reviewing.